drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
@tianon yep -- thanks!  @frapposelli I think it's fine for now.  We have a standardization PR (https://github.com/docker/machine/pull/125) that will use cloudinit and actually does exactly what @tianon is recommending.

thanks for the clarification all!